extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -208,12 +208,47 @@ describe('Project: sqlproj content operations', function (): void {
 		should(projFileText).containEql(convertSlashesForSqlProj(project.getSystemDacpacUri(constants.msdb).fsPath.substring(1)));
 		should(projFileText).containEql(convertSlashesForSqlProj(project.getSystemDacpacSsdtUri(constants.msdb).fsPath.substring(1)));
 
-		await project.addDatabaseReference({ dacpacFileLocation: Uri.file('test.dacpac'), databaseLocation: DatabaseReferenceLocation.sameDatabase });
-		should(project.databaseReferences.length).equal(3, 'There should be three database references after adding a reference to test');
-		should(project.databaseReferences[2].databaseName).equal('test', 'The database reference should be test');
+		// add database reference in the same database
+		await project.addDatabaseReference({ dacpacFileLocation: Uri.file('test1.dacpac'), databaseLocation: DatabaseReferenceLocation.sameDatabase });
+		should(project.databaseReferences.length).equal(3, 'There should be three database references after adding a reference to test1');
+		should(project.databaseReferences[2].databaseName).equal('test1', 'The database reference should be test1');
 		// make sure reference to test.dacpac was added
 		projFileText = (await fs.readFile(projFilePath)).toString();
-		should(projFileText).containEql('test.dacpac');
+		should(projFileText).containEql('test1.dacpac');
+
+		// add database reference to a different database on the same server
```

Comment:
Can you split these database reference tests up into different tests per case?  This unit test is getting pretty heavy as it takes on more cases.